realm/realm-library/src/androidTest/java/io/realm/RealmResultsTest.java

```diff
@@ -1219,4 +1220,85 @@ public void testComparisons() {
         assertEquals(10, testRealm.where(AllTypes.class).lessThanOrEqualTo(AllTypes.FIELD_LONG, -1).findAll().size());
         assertEquals(10, testRealm.where(AllTypes.class).lessThan(AllTypes.FIELD_LONG, 0).findAll().size());
     }
+
+    // RealmResults.distinct(): requires indexing, and type = boolean, integer, date, string
+    private void populateForDistinct(Realm realm, long numberOfBlocks, long numberOfObjects, boolean withNull) {
```

Comment:
Maybe move this to `TestHelper`? But i am fine that do this later when cleaning up `RealmResultsTest`.